app/controllers/api/v1/rent_controller.rb

```diff
@@ -31,6 +39,37 @@ def rent_response
       def rent_serializer
         Api::V1::Serializer::RentSerializer
       end
+
+      def ranking
+        ranking = []
+        Book.all.each do |book|
+          ranking << ranking_response(book)
+        end
+        ranking.sort! { |a, b| b[:times_rented] <=> a[:times_rented] }
+      end
+
+      def ranking_response(book)
+        { id: book.id,
+          author: book.author,
+          title: book.title,
+          image: { url: book.image },
+          publisher: book.editor,
+          year: book.year,
+          genre: book.gender,
+          times_rented: Rent.where(book_id: book.id).count,
+          created_at: book.created_at,
+          updated_at: book.updated_at }
+      end
+
+      def check_active
```

Comment:
Review the `scopes` in ActiveModel 
https://guides.rubyonrails.org/active_record_querying.html#what-is-the-active-record-query-interface-questionmark